tests/ConcordiumNetSdk.UnitTests/Transactions/SendAccountTransactionTests.cs

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
What is this testing? Remove if it is not going to be maintained.

Reply:
This is essentially leftovers from the old repository with a few changes.

I'll add tests today, in a different PR.